Yona/Yona/Extensions.swift

```diff
@@ -20,4 +20,19 @@ extension UIViewController {
         
         return nil
     }
-}
\ No newline at end of file
+}
+
+func setViewControllerToDisplay(value: String, key: String) {
+    let defaults = NSUserDefaults.standardUserDefaults()
+    defaults.setObject(value, forKey: key)
+    defaults.synchronize()
+}
+
+func getViewControllerToDisplay(key: String)-> AnyObject? {
+    let defaults = NSUserDefaults.standardUserDefaults()
+    defaults.objectForKey(key)
+    if defaults.objectForKey(key) == nil {
+    setViewControllerToDisplay("Welcome",key: key)
```

Comment:
Indent for readability, e.g.:

if defaults.objectForKey(key) == nil {
       setViewControllerToDisplay("Welcome",key: key)
}